SendStruct/client.c

```diff
@@ -0,0 +1,72 @@
+#include <arpa/inet.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#define PORT 8080
```

Comment:
We should not use 8080. 8080 is a famous Web port and can be conflict with other programs

Reply:
I changed 8080 to 7473